tools/mpremote/mpremote/main.py

```diff
@@ -449,6 +500,9 @@ def main():
                 do_filesystem(pyb, args)
             elif cmd == "repl":
                 do_repl(pyb, args)
+            elif cmd == "help":
```

Comment:
If this were moved to just after the `if cmd == "connect"` block then it would work even without a connection.